frontend/packages/pipelines-plugin/src/components/pipelines/utils/pipeline-operator.ts

```diff
@@ -13,7 +13,8 @@ export const getPipelineOperatorVersion = async (namespace: string): Promise<Sem
   });
   const matchingCSVs = allCSVs.filter(
     (csv) =>
-      csv.metadata?.name?.startsWith('openshift-pipelines-operator') &&
+      (csv.metadata?.name?.startsWith('redhat-openshift-pipelines') ||
+        csv.metadata?.name?.startsWith('openshift-pipelines-operator')) &&
```

Comment:
From OSP 1.2.x onwards the CSV name is changed from  `openshift-pipelines-operator` to `redhat-openshift-pipelines`, removed `operator` suffix and added `redhat` prefix. I have updated this function so that it can be used in subsequent 1.4 fix related PRs.

Reply:
This code block got merged in my metrics endpoint pr, so I have removed this from this PR to resolve conflicts.